core/components/atoms/dropdown/__stories__/Options.tsx

```diff
@@ -211,3 +236,32 @@ export const fetchOptions = (searchTerm: string, options = dropdownOptions) => {
     }, 1000);
   });
 };
+
+export const fetchMoreThan50DropdownOptions = (searchTerm: string, options = MoreThan50DropdownOptions) => {
+  const searchedOptions = searchTerm ? getSearchedOptions(options, searchTerm) : options;
+  return new Promise<fetchOptionSchema>((resolve) => {
+    window.setTimeout(() => {
+      resolve({
+        searchTerm,
+        options: searchedOptions,
+        count: searchedOptions.length,
+      });
+    }, 1000);
+  });
+};
+
+export const fetchMoreThan50DropdownGroupedOptions = (
```

Comment:
Please use the existing options list or use `selected` prop

Reply:
Done